From 604aa8b1bb2364d30dfe2dc06abbdabc3602a8a8 Mon Sep 17 00:00:00 2001
From: Christine Gharzuzi <cgharzuzi@habana.ai>
Date: Thu, 29 Aug 2019 12:05:14 +0300
Subject: [PATCH] [SW-0] release: add parameters to release job

A three new paramteres are add to the release job
to enable running the job on different devices,
the paramteres have default values in the job.

This commit adds option to override these parameters
using Makefile.

Change-Id: Ie8682a5939d61ba7fabb8f478f5fb6a8558eae48
Signed-off-by: Christine Gharzuzi <cgharzuzi@habana.ai>
---

diff --git a/scripts/make/common/Makefile b/scripts/make/common/Makefile
index f892635..1aa9c31 100644
--- a/scripts/make/common/Makefile
+++ b/scripts/make/common/Makefile
@@ -98,10 +98,14 @@
 RELEASE_JOB_PUBLISH := "publish_artifacts_artifactory=true"
 RELEASE_JOB_RUN_BUILD := "build=true"
 RELEASE_JOB_PROMOTE := "promote=true"
+RELEASE_JOB_HOST_IP := "test_host_ip=$(HL_INSTALL_HOST_IP)"
+RELEASE_JOB_LAB_IP := "test_lab_ip=$(HL_INSTALL_LAB_IP)"
+RELEASE_JOB_SERIAL_PORT := "serial_port=$(HL_DEVICE_SERIAL_PORT)"
 
 RELEASE_JOB_PARAMS := $(RELEASE_JOB_TOKEN)&$(RELEASE_JOB_CAUSE)&$(RELEASE_JOB_BRANCH)&$(RELEASE_JOB_VER_COMPAT)
 RELEASE_JOB_PARAMS := $(RELEASE_JOB_PARAMS)&$(RELEASE_JOB_RUN_TEST)&$(RELEASE_JOB_RUN_LTP)&$(RELEASE_JOB_INSTALL)
 RELEASE_JOB_PARAMS := $(RELEASE_JOB_PARAMS)&$(RELEASE_JOB_PUBLISH)&$(RELEASE_JOB_RUN_BUILD)&$(RELEASE_JOB_PROMOTE)
+RELEASE_JOB_PARAMS := $(RELEASE_JOB_HOST_IP)&$(RELEASE_JOB_LAB_IP)&$(RELEASE_JOB_SERIAL_PORT)
 
 .PHONY: hl-check-branch
 hl-check-branch:
